### ISSUE TITLE
There should be an option to exclude test

### ISSUE THREAD

**Issue by szymonkups** [ckeditor/ckeditor5-dev-builder#15]
We should provide an option to exclude test from compiling process. Projects that will use CKEditor 5 as dependency should have a possibility to compile only source files.

**Comment by pjasiun** [comment]
Maybe there should be an option to **include** tests. Compilation without tests should be a default from the perspective of any other project then `ckeditor5` itself.

**Comment by szymonkups** [comment]
I've created branch with work in progress: https://github.com/ckeditor/ckeditor5-dev-compiler/compare/t/15. There is an option added to `getJsStream` method that allows to exclude tests. I think this will require another config option, let's wait for this https://github.com/ckeditor/ckeditor5/issues/288 to be discussed before introducing another one.

**Comment by Reinmar** [comment]
Does it have to be fixed in it3?

**Comment by pjasiun** [comment]
Well... it's as important as the rest of the tickets I reported recently. It would be nice to have it fixed, but nothing urgent.

**Comment by Reinmar** [comment]
It can be important because it's blocking something. In this case, I assume it's just important cause you don't like it. I don't like it as well, but it doesn't mean that we should waste time on it now.

**Comment by pjasiun** [comment]
Yep, "nothing urgent" was the most important part of my previous post ;)

Anyway: note that testing utils are something, what can be very useful for third-party plugins developers, who do not care about tests building.

**Comment by pjasiun** [comment]
So the tricky part is: should testing utils be built if one does not want to build tests?

**Comment by pjasiun** [comment]
Ok, that's a bigger issue. These tools have absolute paths which do not work with external building systems.

**Comment by jodator** [comment]
@Reinmar It is not a big issue but in my opinion it should be eventually fixed by the time of `1.0.0` milestone.

**Comment by pomek** [comment]
Related ticket: https://github.com/ckeditor/ckeditor5-dev-compiler/issues/49
